packages/frontend/src/components/Routing.js

```diff
@@ -520,7 +520,12 @@ class Routing extends Component {
                             <Route
                                 exact
                                 path="/transfer-wizard"
-                                component={TransferWizardWrapper}
+                                render={() => (
+                                    <TransferWizardWrapper
+                                        account={account}
+                                        onTransfer={this.handleTransferClick} 
```

Comment:
Let's please invoke this in an arrow function to avoid potential binding issues, e.g.:
```js
onTransfer={() => this.handleTransferClick()}
```